# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../postgresql/build_contrib.diff
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./GNUmakefile.in.orig	2011-09-08 23:13:27.000000000 +0200
+++ ./GNUmakefile.in	2011-09-14 13:52:24.616084457 +0200
@@ -8,7 +8,7 @@
 top_builddir = .
 include $(top_builddir)/src/Makefile.global
 
-$(call recurse,all install,src config)
+$(call recurse,world,src config contrib,all)
 
 all:
 	+@echo "All of PostgreSQL successfully made. Ready to install."
